tools/staging_env.sh

```diff
@@ -11,8 +11,8 @@ deactivate() {
     unset deactivate
 }
 
-export UBUNTU_STORE_API_ROOT_URL="https://myapps.developer.staging.ubuntu.com/dev/api/"
-export UBUNTU_STORE_SEARCH_ROOT_URL="https://search.apps.staging.ubuntu.com/"
+export UBUNTU_STORE_API_ROOT_URL="https://dashboard.staging.snapcraft.io/dev/api/"
+export UBUNTU_STORE_SEARCH_ROOT_URL="https://api.snapcraft.io/"
```

Comment:
This should be https://api.staging.snapcraft.io/

Reply:
done